src/sessions/softphone-session-handler.ts

```diff
@@ -100,6 +101,17 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
     this.handleSoftphoneConversationUpdate(update, participant, callState, session);
   }
 
+  holdOtherSessions(currentSession: IExtendedMediaSession): void {
+    const sessions = this.sessionManager.getAllActiveSessions();
+    const otherSessions = sessions.filter(session => session !== currentSession);
```

Comment:
Let's adjust this filter so it's only looking at softphone calls and also only looking for calls that are not already held.